openface/install.bash

```diff
@@ -1,24 +1,23 @@
 #! /usr/bin/env bash
 
-if [ ! -d ~/openface ]; then
-    pushd .
+if [ ! -d ~/openface ]
+then
     mkdir ~/openface
-    cd ~/openface
+    cd ~/openface || tue-install-error "Missing directory: ~/openface"
```

Comment:
`mkdir` and `cd` statements not required as `git clone` is being executed which in fact should be `tue-install-git`

Reply:
`tue-install-git` implemented